README.md

```diff
@@ -48,18 +52,19 @@ $ vagrant up
 
 OR
 
-The app can be run using Docker 
+The app can be run using make commands to run docker 
```

Comment:
Minor, but it would be nice to mention the dependency on make explicitly as we're adding that as the only documented way to run the prod server